resources/test/README.md

```diff
@@ -88,3 +88,8 @@ test may be executed under a number of distinct conditions. These conditions
 are applied using WPT's "test variants" pattern. The available variants are
 defined in the `variants.js` file; this file must be included before
 `testharness.js`. Every test must specify at least one variant.
+
+When the tests are run using `tox`, the actual data for failing tests
```

Comment:
Could you add an ".e.g." here or is that already done elsewhere in this file?

Reply:
`tox` is mentioned above so I think it's the most supported way of running the tests

Reply:
Sorry for being unclear, I meant a way of running a test using tox from the command line.